content/python/concepts/built-in-functions/terms/setattr/setattr.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: 'Setattr()'
+Description: 'Sets the value of the attribute of an object.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Objects'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`Setattr()`** function is a built-in Python function used to set the value of a named attribute of an object. It allows you to dynamically assign or modify attributes of an object at runtime.
+
+## Syntax
+
+```pseudo
+setattr(object, attribute, value)
+```
+
+The `setattr()` function requires three parameters:
+
+- `object`: an object.
+- `attribute`: attribute name that you want to set.
+- `value`: the value you want to give the `attribute`.
```

Comment:
[Proofreading/Nitpick] This is similar to my feedback at line 50. Check it [out](https://github.com/Codecademy/docs/pull/2467/files#r1225934612). Also, using "value" in the description seems redundant. 
```suggestion
- `value`: the value to give the `attribute`.
```